plugins/outputs/groundwork/groundwork.go

```diff
@@ -12,13 +11,14 @@ import (
 	"strings"
 
 	"github.com/gwos/tcg/sdk/clients"
-	"github.com/gwos/tcg/sdk/logper"
+	sdkLog "github.com/gwos/tcg/sdk/log"
 	"github.com/gwos/tcg/sdk/transit"
 	"github.com/hashicorp/go-uuid"
 
 	"github.com/influxdata/telegraf"
 	"github.com/influxdata/telegraf/config"
 	"github.com/influxdata/telegraf/plugins/outputs"
+	logAdapter "github.com/influxdata/telegraf/plugins/outputs/groundwork/slog"
```

Comment:
Please don't add a sub-package but simply add the log-adapter to this package as a separate file e.g. named `log_adapter.go`.

Reply:
Done